src/components/brand/index.tsx

```diff
@@ -0,0 +1,32 @@
+import { clsx } from 'clsx';
+import Image from 'next/image';
+
+export type BrandProps = {
+  theme: 'white' | 'primary';
+};
+
+export default function Brand({ theme = 'primary' }: BrandProps) {
```

Comment:
```suggestion
export default function Brand(props: BrandProps) {
const { theme = 'primary' } = props
```
mình dùng convention này khi dùng props em